README.md

```diff
@@ -110,7 +130,7 @@ Note the use of arrow functions, this is necessary because we are accessing
 
 ## Related
 
-- Focus Trap: https://github.com/davidtheclark/focus-trap
+- Focus Trap: <https://github.com/davidtheclark/focus-trap>
```

Comment:
```suggestion
- Focus Trap: https://github.com/davidtheclark/focus-trap
```

Reply:
This was automatically corrected by Markdownlint ([MD034/no-bare-urls](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md034)). I'll change it back to how it was.